test/onnx/test_fx_type_promotion.py

```diff
@@ -0,0 +1,27 @@
+# Owner(s): ["module: onnx"]
+
+import pytorch_test_common
+from torch.onnx._internal.fx.passes import type_promotion
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class TestGeneratedTypePromotionRuleSet(TestCase):
+    @pytorch_test_common.skip_in_ci(
+        "Reduce noise in CI. "
+        "The test serves as a tool to validate if the generated rule set is current. "
+    )
+    def test_generated_rule_set_is_up_to_date(self):
+        generated_set = type_promotion._GENERATED_ATEN_TYPE_PROMOTION_RULE_SET
+        latest_set = (
+            type_promotion.TypePromotionRuleSetGenerator.generate_from_torch_refs()
+        )
+
+        self.assertEqual(generated_set, latest_set)
+
+    def test_initialize_type_promotion_table_from_generated_set_success(self):
+        type_promotion.TypePromotionTable()
+        self.assertTrue(True)
```

Comment:
It's kind of confusing. Could you add NOTE

Reply:
I can remove L23. the test name describes the expected outcome, but I'll remove `from_generated_set` from the line as it is internal details.